orbit/pkg/scripts/scripts.go

```diff
@@ -59,7 +59,14 @@ func (r *Runner) Run(execIDs []string) error {
 			continue
 		}
 
-		if err := r.runOne(execID); err != nil {
+		script, err := r.Client.GetHostScript(execID)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("get host script: %w", err))
+			// Stop here since we want to preserve the order in which scripts are queued.
+			break
+		}
+
+		if err := r.runOne(script); err != nil {
```

Comment:
Unrelated to the changes but I think we can now safely use `errors.Join` for the code below (lines 74+).